### PR TITLE
Add /info Command and Deploying Dynamically

### PR DESCRIPTION
**Pull request by laupski**

### CODE REVIEW

**Comment by laupski** [comment]
@rmbreak running into issues with async, i think what's happening is that deletes are ongoing at the same time as the creation and nothing is getting created. Thoughts?

File: src/cmd/server.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { RESTGetAPIApplicationGuildCommandsResult, Routes } from 'discord-api-ty
 import { Config, EnvironmentConfig } from '../config';
 import { StockBotSlashCommand } from '../model';
 
-async function createCommands(config: Config) {
+async function createCommands(config: Config, guildId) {
```
Comment (smackysnacks) [inline src/cmd/server.ts:8]: Is `Config.guildId` eventually going to be yanked out? The eventual goal is to have an event listener for when the bot is registered to a new guild, right?

**Review by smackysnacks — DISMISSED** [review]

File: src/cmd/server.ts [hunk 2]
```diff
@@ -64,6 +58,15 @@ async function main(config: Config) {
...
   client.once('ready', () => {
+    client.guilds.valueOf().forEach( guild => {
+      try {
+        deleteCommands(config, guild.id);
+        createCommands(config, guild.id);
+      } catch (error) {
+        console.error(`Could not deploy commands to ${guild.id}! -- ${guild.name}`);
+      }
+    });
+
```
Comment (smackysnacks) [inline src/cmd/server.ts]: Is the intent to delete/create slash commands for each guild sequentially, or do we want to delete/create slash commands concurrently? If the goal is the latter, then we want something like this instead:

```suggestion
  client.once('ready', async () => {
    await Promise.all(client.guilds.valueOf().map( async guild => {
      try {
        await deleteCommands(config, guild.id);
        await createCommands(config, guild.id);
      } catch (error) {
        console.error(`Could not deploy commands to ${guild.id}! -- ${guild.name}`);
      }
    }));
```

File: src/cmd/server.ts (hunk 1 above)
Comment (laupski) [reply src/cmd/server.ts:8]: Correct, i was gonna do a follow up commit removing `Config.guildId`

**Review by smackysnacks — APPROVED** [review]

**Pull request merged**